src/main/java/com/basho/riak/client/core/converters/TimeSeriesConverter.java

```diff
@@ -0,0 +1,203 @@
+package com.basho.riak.client.core.converters;
+
+import com.basho.riak.client.core.query.timeseries.Cell;
+import com.basho.riak.client.core.query.timeseries.ColumnDescription;
+import com.basho.riak.client.core.query.timeseries.QueryResult;
+import com.basho.riak.client.core.query.timeseries.Row;
+import com.basho.riak.client.core.util.BinaryValue;
+import com.basho.riak.protobuf.RiakKvPB;
+import com.google.protobuf.ByteString;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * @author Alex Moore <amoore at basho dot com>
+ * @since 2.0.3
+ */
+public class TimeSeriesConverter
```

Comment:
To reduce memory consumption, since it is stateless and without polymorphic behavior, better to use as a kind of static facade.